Source/Core/Core/IOS/ES/Formats.cpp

```diff
@@ -44,7 +44,8 @@ bool IsChannel(u64 title_id)
 {
   return IsTitleType(title_id, TitleType::Channel) ||
          IsTitleType(title_id, TitleType::SystemChannel) ||
-         IsTitleType(title_id, TitleType::GameWithChannel);
+         IsTitleType(title_id, TitleType::GameWithChannel) ||
+         IsTitleType(title_id, TitleType::HiddenChannel) || title_id == TITLEID_SYSMENU;
```

Comment:
If I put it first, then the first IsTitleType call is not aligned with the others :( (clang-format forces the IsTitleType to be on the same line as the system menu title ID check)

I have moved the title ID to its own if conditional.